packages/pangea-node-sdk/tests/integration/intel.test.ts

```diff
@@ -24,6 +27,15 @@ it("file lookup should succeed", async () => {
   expect(response.result.data.verdict).toBe("malicious");
 });
 
+it("file lookup with default provider should succeed", async () => {
+  const response = await fileIntel.lookup(
+    "142b638c6a60b60c7f9928da4fb85a5a8e1422a9ffdc9ee49e17e56ccca9cf6e",
```

Comment:
same comment here as in the Java MR, do we want to make these globals to the integration tests?

Reply:
@doubletooth Honestly, I much prefer seeing the value directly within the test (how its defined here) instead of hunting for a global definition.

Reply:
if it's only used once (like this one is) I'm okay with that. I do believe the IP is used in a bunch of places though

Reply:
Even for IP I don't think it should be a global.

This is the philosophy I strongly agree with in tests: https://thoughtbot.com/blog/the-self-contained-test

> One of the earliest pieces of wisdom we are given as programmers is to not write duplicate code: Don’t Repeat Yourself (or DRY if you prefer). Identical blocks of code to set up a test sure does look like repetition, so we extract it into [...]
>
> This is a mistake for tests.
>
> Optimize for understanding, not code brevity

Reply:
I don't feel very strongly about this, we can keep it within the test

Reply:
@doubletooth I get what you say now, I didn't get on Java MR. Yes, I think we can use a global variable, particularly in Java we have a lot of instances because we have more methods to tests.

Reply:
I'm fine either way @pangea-andrest, so I'll resolve this thread